### PR TITLE
Release 0.15.1

### PR DESCRIPTION
**Pull request by ni4**
This is mostly bugfix release, so changelog is rather short. Feel free to suggest changes or updates if something was omitted.

### CODE REVIEW

**Review by antonsviridenko — APPROVED** [review]

**Comment by ni4** [comment]
Please do not merge once approved, as question with #1510 is still open.

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Looks good, thank you @ni4 !

**Pull request merged**
